5.5/apache/Dockerfile

```diff
@@ -27,6 +27,15 @@ RUN a2enmod rewrite headers
 COPY magento.conf /etc/apache2/conf-enabled/
 
 
+ENV ION_CUBE_PHP_VERSION "5.5"
+ENV PHP_EXTENSION_DIR "/usr/local/lib/php/extensions/no-debug-non-zts-20121212"
```

Comment:
You can't do dynamic builds in docker.. so I tried to use `ENV PHP_EXTENSION_DIR "$(php -r 'echo ini_get("extension_dir");'`)" which doesn't work :/ spent a fair while trying to do that yesterday! http://stackoverflow.com/questions/27711304/create-dynamic-environment-variables-at-build-time-in-docker